optaweb-vehicle-routing-backend/pom.xml

```diff
@@ -234,51 +214,21 @@
         <executions>
           <execution>
             <id>validate</id>
+            <phase>none</phase>
+            <configuration>
+              <skip>true</skip>
+            </configuration>
+          </execution>
+          <execution>
+            <id>validate-override</id>
```

Comment:
Could you please also remove the override execution when finishing the PR? Since we use `combine.self="override"` on the `<configuration>` element, it's no longer needed to disable the original `validate` execution and create a new one.